topics/go/algorithms/strings/palindrome/palindrome_test.go

```diff
@@ -0,0 +1,35 @@
+package strings_test
+
+import (
+	strings "github.com/ardanlabs/gotraining/topics/go/algorithms/strings/palindrome"
+	"testing"
+)
+
+const succeed = "\u2713"
+const failed = "\u2717"
+
+func TestIsPalindrome(t *testing.T) {
+
+	revTests := []struct {
+		name     string
+		input    string
+		expected bool
+	}{
+		{"empty string", "", true},
+		{"string with length of 1", "G", true},
+		{"string with odd length", "bob", true},
+		{"string with even length", "otto", true},
+		{"chinese", "汉字汉", true},
+		{"failed test", "test", false},
+	}
+
+	for _, tt := range revTests {
+		got := strings.IsPalindrome(tt.input)
+		if got {
```

Comment:
Try to stay away from `else`. You can use a continue in the `if` to do that.

Reply:
Done